scripts/create_workspace.py

```diff
@@ -0,0 +1,26 @@
+# this script is called by AzureML notebooks to create workspace config file for the first run 
```

Comment:
it's curious, don't you get an error for not importing os and Workspace?

Reply:
Because this code is meant to be injected to a cell in sar_movielens_with_azureml notebook, where Workspace and OS are already imported